### ISSUE TITLE
A program problem

### ISSUE THREAD

**Issue by mumu029** [ebagdasa/backdoor_federated_learning#18]
It seems to me that poison_dataset() didn't poison the data, it just sampled 64 images 200 times and required them not to be images from "posion_image" and "poison_images_test". But what does that have to do with data poisoning.

**Comment by RiverCrosser-White** [comment]
i agree with you, i think maybe some related code was commented out.

**Comment by YH-star1** [comment]
Without finding the real poisoning of the data, I guess it is possible that the author used the poisoning method of "using the picture itself as a trigger", but I have little experience, just guess.
